src/components/AddToList.vue

```diff
@@ -1,11 +1,47 @@
 <script setup>
+import { ref } from 'vue';
+const emit = defineEmits(['add']);
+const name = ref('');
+const giftsList = ref([]);
+const naughty = ref(false);
+
+const handleSubmit = () => {
+    emit('add',addname.value, giftsList.value, naughty.value);
+}
 
 </script>
 
 <template>
-
+    <form id="addToList" @submit.prevent="handleSubmit">
+        <h2>Add to the list</h2>
+        <input v-model="name" type="text" placeholder="Your name" required></input>
+        <input v-model="giftsList" type="text" placeholder="Add your gift name" required></input>
```

Comment:
We should be able to add multiple gifts if giftsList is an array. Let's do in next sprint.